src/lib/renderer.ts

```diff
@@ -89,7 +89,8 @@ export function generateHtml(
     </vertex-viewer>
 
     <script type="module">
-      import { applyPolyfills, defineCustomElements } from 'https://unpkg.com/@vertexvis/viewer@0.9.x/dist/esm/loader.js';
+      import { applyPolyfills } from 'https://unpkg.com/@vertexvis/viewer@0.9.x/loader/index.js';
+      import { defineCustomElements } from 'https://unpkg.com/@vertexvis/viewer@0.9.x/dist/esm/loader.js';
```

Comment:
You should be able to pull `defineCustomElements` from https://unpkg.com/@vertexvis/viewer@0.9.x/loader/index.js as well.